MemberExpectations.md

```diff
@@ -0,0 +1,60 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+It is understood that members will have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
```

Comment:
Suggest: `It is understand that members will have and will express their individual opinions during discussions within the organization` ... there's no reason to `encourage` what should already be understood.

Reply:
addressed